examples/onft721-zksync/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@layerzerolabs/onft721-zksync-example",
+  "version": "0.2.0",
+  "license": "MIT",
```

Comment:
please add `"private": true,` above this line.  We do not actually want to publish this to NPMJS (no one should depend directly on the example as an NPM dependency).